### ISSUE TITLE
Core instruction issue.

### ISSUE THREAD

**Issue by MateoConLechuga** [CE-Programming/CEmu#35]
For some odd reason, this executes incorrectly inside the core: (Set a breakpoint at D1AC18 to see)

The instruction in question is 'ld ix,6'. However, it executes as 'ld hl,6' in this context.

To test it, run the program `Asm(pgrmGRAPHICS`

[CalcImage](http://myimages.wikidot.com/local--files/start/CEmu_ix_bug.ce)

Image:
<a href="http://i.imgur.com/5eCG5UW.png"><img src="http://i.imgur.com/5eCG5UW.png" /></a>

**Comment by Runer112** [comment]
That instruction executes properly for me. However, I'm a little worried that the rest of the program may not have executed correctly... should the output look like this?

![graphics](https://cloud.githubusercontent.com/assets/6608979/13300172/ce318bbe-db0c-11e5-9818-b9f9f0f6ab6f.png)

For the record, this seems very related to a bug I created during development of the block-instructions branch. But I was fairly sure it was fixed before the branch was merged into master (2fd34e849757bd8e085a694be63fae79eae7bc9c).
